node-tests/acceptance/exam-test.js

```diff
@@ -414,7 +441,7 @@ describe('Acceptance | Exam Command', function() {
           'failure-dist',
           '--load-balance',
           '--parallel',
-          '2',
+          '1',
```

Comment:
curious, whats the reason for this change?

Reply:
I've made changes because it seems to create multiple test-execution file from multiple events are called and `browserExitHandler` is executed multiple times. After I remove `disconnect` callback registering I made changes back to test against 2 browsers.